onnxruntime/test/optimizer/graph_transform_test_layernorm.cc

```diff
@@ -429,6 +429,38 @@ TEST_F(GraphTransformationTests, SimplifiedLayerNormFusionTest) {
   }
 }
 
+TEST_F(GraphTransformationTests, LayerNormScaleBiasTest) {
+  constexpr const ORTCHAR_T* model_uri = MODEL_FOLDER "fusion/layer_norm_fusion_scale_bias.onnx";
```

Comment:
shall we give more details explainationg what the test is testing?

Reply:
for example: test the case when scale or bias are not graph input or graph initilizer.